modules/osd-aws-privatelink-architecture.adoc

```diff
@@ -1,21 +1,23 @@
 [id="osd-aws-privatelink-architecture.adoc_{context}"]
 = PrivateLink architecture
 
-The Red Hat managed infrastructure responsible for creating PrivateLink clusters is hosted on completely private subnets. The only connection made between Red Hat and the customer provided infrastructure is through PrivateLink VPC Endpoints. 
+The Red Hat managed infrastructure that creates AWS PrivateLink clusters is hosted on completely private subnets. The only connection made between Red Hat and the customer provided infrastructure is through PrivateLink VPC Endpoints.
```

Comment:
Remove _completely_ Also consider this edit: The connection between Red Hat and the customer-provided infrastructure is created through PrivateLink VPC endpoints.